### ISSUE TITLE
Password type

### ISSUE THREAD

**Issue by bayfatih2000** [adib2149/FormMaster#29]
I downloaded the example and run it but I can see the password. I want to see as stars (*****). 
How can I do it?

**Comment by skylord83** [comment]
write this to the end of  FormElementTextPasswordViewHolder constructor:

_mEditTextValue.setTransformationMethod(new PasswordTransformationMethod());_

**Comment by nishanthoblirajan** [comment]
Is it working? how to implement it?

**Comment by skylord83** [comment]
yes it works fine. You can follow the steps in the installation guide or you can download and add it as a library if you want to customize it.
